build-tools/xaprepare/xaprepare/Application/Abi.cs

```diff
@@ -26,12 +26,12 @@ partial class Abi
 			new Abi { Name = AbiNames.HostJit.Win32,           Type = AbiType.HostJit,   Is64Bit = false, IsCross = false, IsHost = true,  IsLlvm = false, IsWindows = true },
 			new Abi { Name = AbiNames.HostJit.Win64,           Type = AbiType.HostJit,   Is64Bit = true,  IsCross = false, IsHost = true,  IsLlvm = false, IsWindows = true },
 
-			new Abi { Name = AbiNames.CrossAot.ArmV7a,         Type = AbiType.CrossAot,  Is64Bit = false, IsCross = true,  IsHost = false, IsLlvm = false, IsWindows = false },
```

Comment:
These changes shouldn't be needed as of bb7ffb775599bdb4675301ae06d1bbc8e8ff8566 and https://github.com/xamarin/xamarin-android/blob/aef33486566f117f0089ea5f4cbebc37aa37ff95/build-tools/xaprepare/xaprepare/Application/MonoCrossRuntime.cs#L21

Please revert the changes to this file.